tests/feature_tests/ci/test.sh

```diff
@@ -14,7 +14,7 @@ python3 -m venv venv
 . venv/bin/activate
 ci/pip-install.sh
 coverage erase
-RENDERER_HOST=localhost python tests/quality_time_api_server_under_coverage.py &> ../../build/quality_time_api_server.log &
+env RENDERER_HOST=localhost DATABASE_HOST=localhost LDAP_URL=ldap://localhost:389 python tests/quality_time_api_server_under_coverage.py &> ../../build/quality_time_api_server.log &
```

Comment:
Why `env` ?

Reply:
I wasn't sure the environment variables were influencing the later docker compose call. Now running the tests without `env`. I'll remove it, if possible.